examples/cd_spot_the_diff_mnist_wine.ipynb

```diff
@@ -48,16 +48,27 @@
    "cell_type": "code",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    all_x, all_y = mnist_train_ds.train_data, mnist_train_ds.target
</pre><p> </p><p>GIves:</p><pre class="ql-syntax" spellcheck="false">`AttributeError: 'MNIST' object has no attribute 'target'
</pre><p> </p><p>Possibly  a <code>torchvision</code>  versioning issue (I'm on <code>0.10.0+cpu</code>).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Correction: Above is nothing to do with `torchvision` versioning, just a typo. 

`all_x, all_y = mnist_train_ds.train_data, mnist_train_ds.target`

should be:

`all_x, all_y = mnist_train_ds.train_data, mnist_train_ds.targets`

Reply:
Ah, I did see that and thought I fixed it already.